dev/Expander/Expander.xaml

```diff
@@ -128,36 +126,36 @@
                             contract7Present:BackgroundSizing="{TemplateBinding BackgroundSizing}"
-                            BorderBrush="{TemplateBinding BorderBrush}"
-                            BorderThickness="{TemplateBinding BorderThickness}"
+                            BorderBrush="{ThemeResource ExpanderHeaderBorderBrush}"
+                            BorderThickness="{ThemeResource ExpanderHeaderBorderThickness}"
                             MinHeight="{TemplateBinding MinHeight}"
-                            MinWidth="{TemplateBinding MinWidth}"
-                            MaxWidth="{TemplateBinding MaxWidth}"
                             contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
                             IsEnabled="{TemplateBinding IsEnabled}"
-                            Padding="{TemplateBinding Padding}"
+                            Padding="{StaticResource ExpanderHeaderPadding}"
                             Style="{StaticResource ExpanderHeaderDownStyle}"
+                            HorizontalAlignment="Stretch"
+                            HorizontalContentAlignment="{ThemeResource ExpanderHeaderHorizontalContentAlignment}"
+                            VerticalContentAlignment="{ThemeResource ExpanderHeaderVerticalContentAlignment}"
                             Content="{TemplateBinding Header}"
                             ContentTemplate="{TemplateBinding HeaderTemplate}"
                             ContentTemplateSelector="{TemplateBinding HeaderTemplateSelector}"
-                            HorizontalAlignment="{TemplateBinding HorizontalAlignment}"
                             IsChecked="{Binding Path=IsExpanded, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}" />
                         <!-- The clip is a composition clip applied in code -->
                         <Border x:Name="ExpanderContentClip" Grid.Row="1">
                             <Border
-                                x:Name="ExpanderContent"                         
+                                x:Name="ExpanderContent"
                                 Visibility="Collapsed"
-                                Background="{ThemeResource ExpanderDropDownBackground}"
+                                Background="{TemplateBinding Background}"
                                 contract7Present:BackgroundSizing="{TemplateBinding BackgroundSizing}"
                                 contract7Present:CornerRadius="{Binding CornerRadius, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource BottomCornerRadiusFilterConverter}}"
-                                BorderBrush="{ThemeResource ExpanderDropDownBorderBrush}"
-                                BorderThickness="{StaticResource ExpanderDropdownDownBorderThickness}"
+                                BorderBrush="{TemplateBinding BorderBrush}"
+                                BorderThickness="{StaticResource ExpanderContentDownBorderThickness}"
```

Comment:
This is not ideal.
 - `BorderThickness="{StaticResource ExpanderContentDownBorderThickness}"` should be
 - `BorderBrush="{TemplateBinding BorderThickness}"`

However, that would not work due to the `BorderThickness` of the control having local precedence (higher precedence than the visual state setter). The visual state is being used to switch between border thicknesses that have the top or bottom edge set to zero depending on the expand direction.

The correct solution to this is to add another filter converter for the border thickness similar to the CornerRadius. I expect push back so I didn't implement that.

What is done now is considered a hack but is usable for an initial release. Developers can still customize the thickness of the content area using `ExpanderContentDownBorderThickness` and `ExpanderContentUpBorderThickness` even while the `BorderThickness` of the control is ignored.